perf.groovy

```diff
@@ -27,8 +27,22 @@ def static getOSGroup(def os) {
 // Setup perflab tests runs
 [true, false].each { isPR ->
     ['Windows_NT'].each { os ->
-		['x64', 'x86'].each { architecture ->
-			def newJob = job(Utilities.getFullJobName(project, "perf_perflab_${os}_${architecture}", isPR)) {
+		['x64', 'x86ryujit', 'jit32'].each { arch ->
+            architecture = arch
+            testEnv = ''
+
+            if (arch == 'jit32')
+            {
+                architecture = 'x86'
+                testEnv = '-testEnv %WORKSPACE%\\tests\\x86\\compatjit_x86_testenv.cmd'
+            }
+            else if (arch == 'x86ryujit')
```

Comment:
@DrewScoggins, I asked for this to make it easier for us to swap out the JITs.  We'd like to be able to (more) atomically make either JIT the default and not affect these perf jobs.  By putting the logic that determines which JIT is used in external scripts instead of the groovy, it is easier to validate that we did that correctly.